attacker/apis.go

```diff
@@ -12,7 +12,7 @@ import (
 // It returns a url and headers for the specified input.
 func getRequestCommons(ctx context.Context, endpoint, host, token string) (string, http.Header) {
 	url := host + endpoint
-	zlog.Debug(ctx).Str("endpoint", url).Msg("preparing headers")
+	zlog.Info(ctx).Str("endpoint", url).Msg("preparing headers")
```

Comment:
A lot of these seem like genuine `Debug()` messages, if QE wants to see this output, can they just run with log level `debug`?

Reply:
I changed it to `Info` as I felt like there are no enough logs when enabled `Info` as log level. I am thinking to keep these as `Info` and enable `Debug` only in case of error messages. Does that sound okay?